packages/sanity/src/core/versions/components/dialog/CreateBundleDialog.tsx

```diff
@@ -1,75 +1,71 @@
 import {ArrowRightIcon} from '@sanity/icons'
 import {Box, Button, Dialog, Flex} from '@sanity/ui'
-import {useCallback, useState} from 'react'
-import {useCurrentUser} from 'sanity'
+import {type FormEvent, useCallback, useState} from 'react'
 
+import {type BundleDocument} from '../../../store/bundles/types'
 import {useBundleOperations} from '../../../store/bundles/useBundleOperations'
-import {type Bundle} from '../../types'
-import {getRandomToneIcon, isDraftOrPublished} from '../../util/dummyGetters'
+import {isDraftOrPublished} from '../../util/dummyGetters'
 import {BundleForm} from './BundleForm'
 
-export function CreateBundleDialog(props: {
-  onCancel: () => void
-  onCreate: (value: Bundle) => void
-}): JSX.Element {
-  const {onCancel, onCreate} = props
+export function CreateBundleDialog(props: {onClose: () => void}): JSX.Element {
```

Comment:
`BundlesOverview` will need its props updated

Reply:
Alright, making this change :)